h/models/group_scope.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import sqlalchemy as sa
+
+from h._compat import urlparse
+from h.db import Base, mixins
+
+
+class GroupScope(Base):
+    __tablename__ = 'groupscope'
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+
+    #: A fully qualified domain name, e.g. example.com, www.nytimes.com or
+    #: web.hypothes.is.
+    hostname = sa.Column('hostname', sa.UnicodeText)
+
+    groups = sa.orm.relationship(
+        'Group',
+        secondary='groupscope_group',
+        backref=sa.orm.backref('scopes', order_by='Group.name'),
+    )
+
+    def __repr__(self):
+        return '<GroupScope %s>' % self.id
+
+
+GROUP_GROUPSCOPE_TABLE = sa.Table('groupscope_group', Base.metadata,
```

Comment:
Just doublechecking vocabulary here. `scope`, and `group_scope` as tablenames would, I presume, seem too general-purpose?

Reply:
Also, it's `GROUP_GROUPSCOPE_TABLE` but the tablename is `groupscope_group`. Is that intentional/a pattern?

Reply:
Yeah, I went for `groupscope` instead of just `scope` because I thought `scope` sounded too general, as these are only for groups.